xandikos/icalendar.py

```diff
@@ -66,9 +66,9 @@ def calendar_prop_delta(old_component, new_component):
     for field in fields:
         old_value = old_component.get(field)
         new_value = new_component.get(field)
-        if (getattr(old_value, 'to_ical', None) is None or
-            getattr(new_value, 'to_ical', None) is None or
-            old_value.to_ical() != new_value.to_ical()):
+        if getattr(old_value, 'to_ical', None) is None or \
```

Comment:
Please keep the original formatting here; PEP8 prefers parentheses:

"The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation."